### DISCUSSION TITLE
Feature request: office productivity software

### DISCUSSION THREAD

**Discussion by adorokhine** [DustinBrett/daedalOS#407] [Feature Requests]
This project is really cool! It would be very useful to have productivity software (word processor, slides, spreadsheet).
Currently there only seems to be a text editor, and Google Drive doesn't seem to run in the embedded browser.

**Comment by DustinBrett** [comment]
Thanks for the suggestion and I totally agree. I have a long term plan to add Google Drive and OneDrive support. The main things holding me back from it is that I don't desire to get/require an api key for features, if I can avoid it. As for a spreadsheet viewer/editor, I have wanted to add one for a while but it's been a lower priority. I also like the idea of running the WASM version of OpenOffice, but it's rather big and slow. I will keep looking into options and I think eventually I will have more robust productivity tools.

**Reply by NamelessGO** [reply]
> running the WASM version of OpenOffice, but it's rather big and slow.

[LibreOffice](https://www.libreoffice.org/) is the successor to OpenOffice, which had its last major update in 2014
https://www.libreoffice.org/discover/libreoffice-vs-openoffice/

There is also [OnlyOffice](https://www.onlyoffice.com/) another FOSS project built upon OOXML to provide the best compatibility with MS Office

**Reply by DustinBrett** [reply]
Ya for sure whatever is latest would be ideal. But in the wasm world at the moment I have not been picky. I hope one day to learn more about porting and do my own.
The WASM build I did recall actually was Libra though so that is good, but an older version.

https://lab.allotropia.de/wasm/
